eland/ml/pytorch/_pytorch_model.py

```diff
@@ -88,12 +89,14 @@ def model_file_chunk_generator() -> Iterable[str]:
     def import_model(
         self,
         model_path: str,
-        config_path: str,
```

Comment:
This would technically be a break as `PyTorchModel` is a public facing API. We could make the `import_model` method keyword only, have both `config_path` and `config` as parameters, and emit a `DeprecationWarning` if `config_path` is used as a way to use `config` in the future?

Reply:
Do we have any rules around tech-preview features? The APIs being used in Elasticsearch are techpreview and subject to change. Seems like the client objects interacting with them should have the same bwc guarantees no?